Gemfile.lock

```diff
@@ -396,13 +391,9 @@ GEM
     spring-watcher-listen (2.0.1)
       listen (>= 2.7, < 4.0)
       spring (>= 1.2, < 3.0)
-    sprockets (3.7.2)
+    sprockets (4.0.0)
```

Comment:
### [sprockets](https://github.com/rails/sprockets)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 3.7.2 → 4.0.0

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **881** commits. See the full changes on [the compare page](https://github.com/rails/sprockets/compare/v3.7.2...v4.0.0).

These are the first 10 commits:
- (a4001a4) [Fixnum and Bignum are deprecated on Ruby trunk](https://github.com/rails/sprockets/commit/a4001a4b2f8408f0a87ff44aa21b502c1847f79e)
- (bab65a6) [Merge pull request #392 from matthewd/fixnum-warning](https://github.com/rails/sprockets/commit/bab65a689ba3bd8e4bbf3369343cef2d53f97c57)
- (f137665) [Appends load paths to error message with tests](https://github.com/rails/sprockets/commit/f1376657c3d55aaf5372586c5029881cbb6f6793)
- (5987dd3) [Fix tests](https://github.com/rails/sprockets/commit/5987dd3be8300e76947f4d06ebbc9dc6817cbe88)
- (2a5ad6f) [Merge pull request #322 from rails/schneems/load-paths-on-er…](https://github.com/rails/sprockets/commit/2a5ad6fe44d965b9f63fee727150f777f2d8f6ba)
- (1cf6076) [Revert "Add env dependency resolver"](https://github.com/rails/sprockets/commit/1cf6076116592593329649cf0bbf8c3772ed4195)
- (55fbb54) [Merge pull request #396 from rails/revert-365-env-dep-resolv…](https://github.com/rails/sprockets/commit/55fbb5464b2acca5d50cacef28766cf61aaac84e)
- (1275e40) [Fix test name](https://github.com/rails/sprockets/commit/1275e4024f69517bd244816ccf003b0ee8dd5f18)
- (5478dbf) [Failure case for #394](https://github.com/rails/sprockets/commit/5478dbf9e903d189af48149caf22cc9663db755f)
- (060b5c0) [Merge pull request #397 from rails/schneems/test-top-level-c…](https://github.com/rails/sprockets/commit/060b5c0b7d86708ca7c3ae50a39cc07dfe64fdb3)


</details>